.github/workflows/dockerhub-description.yml

```diff
@@ -0,0 +1,78 @@
+
+
+name: Update Docker Hub Description
+on:
+  push:
+    branches: [ develop, main ]
+    paths: [ README.md, .github/workflows/dockerhub-description.yml ]
+  pull_request:    
+    branches: [ develop, main ]
+    paths: [ README.md, .github/workflows/dockerhub-description.yml ]
+  
+
+  
+jobs:
+  # There is no support for yaml anchor on Github_actions
+  # https://github.com/actions/runner/issues/1182 
+  dockerHubDescription:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Docker Hub Description agent
+      uses: peter-evans/dockerhub-description@v2
+      # The default readme is in the root of the repository but any file 
+      # can be used, example: readme-filepath: ./path/to/README.md
+      with: 
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
can we change this to DOCKERHUB_TOKEN? that is already defined.

Reply:
Sorry I meant the `secrets` name for `password` is `DOCKERHUB_TOKEN` ... so:
```
        password: ${{ secrets.DOCKERHUB_TOKEN }}
```
I would expect that to work, are you saying it wouldn't?

Reply:
I'll dig more on this.